tests/shell/shell_recorder/mathcheck.esr

```diff
@@ -5,7 +5,7 @@ Storage: mathcheck
 
 # Setup
 
-< kdb mount $File $Mountpoint $Storage
+< kdb mount $File $Mountpoint $Storage dump
```

Comment:
It would be better if we can get rid of the dump plugin. Are there problems with the meta data within the INI plugin?

Reply:
I do not know what the exact problem is, but I would assume it has something to do with metadata. The test:

```sh
kdb mount mathcheck.esr user/tmount/mathcheck mathcheck

kdb set user/tmount/mathcheck/check/test0/points 5.9
kdb set user/tmount/mathcheck/check/test1/points 4.5

kdb set user/tmount/mathcheck/check/gp
kdb setmeta user/tmount/mathcheck/check/gp check/math ":= + ../test0/points ../test1/points"

kdb get user/tmount/mathcheck/check/gp
# STDOUT: 10.4
```

simply does not work if we use the INI as default storage plugin. I substituted `dump` with `ni` in the latest update.

Reply:
Seems to be a bug, could you please report it with the logs?